todo_project/settings/staging.py

```
@@ -71,3 +71,9 @@
 SESSION_COOKIE_DOMAIN = ".realdevsquad.com"
 SESSION_COOKIE_SAMESITE = "None"
 CSRF_COOKIE_SECURE = True
+
+SPECTACULAR_SETTINGS.update({
+    "SWAGGER_UI_SETTINGS": {
+        "url": "/staging-todo/api/schema",
+    },
+})
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)